test/e2e/SKIPPED_TESTS.yaml

```diff
@@ -68,11 +61,20 @@ skipped_tests:
 - TestTinkerbellKubernetes130RedHatSimpleFlow
 - TestTinkerbellKubernetes126UbuntuSimpleFlow
 - TestTinkerbellKubernetes127UbuntuSimpleFlow
+- TestTinkerbellKubernetes128UbuntuSimpleFlow
+- TestTinkerbellKubernetes129UbuntuSimpleFlow
```

Comment:
I think we still run some older version simpleFlow/upgrade tests to catch any regression that could be specific to a kube version. For example, we found a regression on InPlace that only impacted 1.27 version and not others. Are we skipping this only to reduce the e2e runtime?

Reply:
Since we already skip the tests for oldest versions(1.26 and 1.27), I think it makes sense to skip it for 1.28 and 1.29 too with that reasoning. But if we are worried about catching any regressions with a particular kubernetes version, then it would also apply to those older versions which are already skipped. So I feel we should either skip all the older versions and just run the latest one or don't skip any of them at all.

Reply:
I think we should try running at least the simple tests for all Kubernetes version to catch any version specific regression! So let's unskip the older two versions simple flow tests